### DISCUSSION TITLE
Help with converting tile-layer-non-geospatial Moon example to Geospatial

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5610] [Q&A]
Hi, I would like to try to convert the example [tile-layer-non-geospatial](https://deck.gl/examples/tile-layer-non-geospatial/) moon example into a geospatial example (as practice for working with my own deepzoom images in geospatial views). Would I need to modify the deepzoom tiles or the .dzi file in order to get the moon to render in geospatial format? I have these examples on Observable
* [non-geospatial moon example ported to Observable](https://observablehq.com/@cornhundred/deck-gl-moon-tillayer-example?collection=@cornhundred/deck-gl)
* [attempt to convert to geospatial](https://observablehq.com/d/f6c28b30d9cdae2e?collection=@cornhundred/deck-gl)

I'm currently seeing errors about deck.gl looking for the wrong tiles probably because the bounding box is not being set correctly

<img width="193" alt="Screen Shot 2021-03-23 at 2 03 00 PM" src="https://user-images.githubusercontent.com/8352840/112195609-8bd72500-8be0-11eb-8e3f-d9e0500af40f.png">

**Comment by kylebarron** [comment]
You can use `getTileData` to load whatever tile you need. It exposes a `bbox` argument. But your tiles need to be in web mercator to be used with a geospatial view.

**Reply by cornhundred** [reply]
Thanks @kylebarron. Would you happen to know if I can still use vips to make the web mercator tiles or is there a different software that is better for this?

**Reply by cornhundred** [reply]
I'm going to go through this tutorial from mapbox https://docs.mapbox.com/help/tutorials/georeferencing-imagery/ since it seems to be similar to what I would like to do
